### ISSUE TITLE
QR Codes without a border are not recognized

### ISSUE THREAD

**Issue by davidarmstronglewis** [piderman314/bardecoder#8]
Images generated by [qrcode](https://github.com/kennytm/qrcode-rust), seems that if I include a border all is good, if not, bardecoder is unable to recognize the code.

Here is the broken image:
![broken](https://user-images.githubusercontent.com/6754950/63562750-a7b1ab00-c513-11e9-93b7-be67b268da05.png)

As well as the working one:
![working](https://user-images.githubusercontent.com/6754950/63562757-ab453200-c513-11e9-96a8-1220ffea1f8b.png)

Here's the code I'm using, pretty much copy-pasta from the README :)
```rust
let decoder = bardecoder::default_decoder();
let results = decoder.decode(image::open("./output/broken.png")?);
println!("{}", results.len());
for r in results {
  println!("{:#?}", r);
}
```

**Comment by piderman314** [comment]
Looks like you are right. I'm going to investigate :)

**Comment by piderman314** [comment]
This has been fixed in version 0.1.2 . Thanks again for the report!

**Comment by davidarmstronglewis** [comment]
Thank you!
